hr_holidays_public/wizards/hr_holidays_public_generator.py

```diff
@@ -0,0 +1,85 @@
+# Copyright 2018 elego Software Solutions GmbH - Yu Weng
+# Copyright 2018 initOS GmbH - Nikolina Todorova
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+from datetime import datetime
+
+from odoo import _, api, fields, models
+from odoo.exceptions import UserError
+
+COUNTRY_GENERATORS = []
+
+
+class HrHolidaysPublicGenerator(models.TransientModel):
```

Comment:
I think the computation of easter sunday should be in this class : otherwise most western european localizations will end up recomputing it.

Reply:
Yes, It's a good Idea. I will try to improve this wizard in next PR.